tensorflow/core/graph/mkl_layout_pass_test.cc

```diff
@@ -1341,6 +1341,288 @@ TEST_F(MklLayoutPassTest, NodeMerge_TransposeConv3DTranspose_Negative) {
             "Transpose1:control->DMT/_2:control");
 }
 
+TEST_F(MklLayoutPassTest, NodeMerge_TransposeMaxPool3DTranspose_Positive) {
+  InitGraph(
+      "node { name: 'Input0' op: 'Input'}                                     \
+       node { name: 'Input1' op: 'Input'}                                     \
```

Comment:
It doesn't look like we need `Input1` here.

Reply:
Will remove it.